devices/philips.js

```diff
@@ -852,6 +861,15 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp_color(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['4090130P9'],
+        model: '4090130P9',
+        vendor: 'Philips',
+        description: 'Hue Sana',
+        meta: {turnsOffAtBrightness1: true},
+        extend: hueExtend.light_onoff_brightness_colortemp_color(),
```

Comment:
Please add the color temp range for both, see https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html#31-retrieving-color-temperature-range-only-required-for-lights-which-support-color-temperature

Reply:
Both have 153-500. I think it should be added now.